scripts/V326__CARGA_QUESTIONARIO_ENCAMINHAMENTO_AEE_ETAPA_2.sql

```diff
@@ -0,0 +1,47 @@
+do $$
+declare 
+	questionarioId bigint;
+	questaoId bigint;
+	questaoComplementarId bigint;
+	questaoComplementar2Id bigint;
+	
+begin
+	/********************************/
+	/* Questionário Seção 3 Etapa 2 */
+	/********************************/
+	insert into questionario (nome, excluido, criado_em, criado_por, criado_rf)
+		values ('Questionário Encaminhamento AEE Etapa 2 Seção 3', false, NOW(), 'SISTEMA', '0')
+		RETURNING id INTO questionarioId;
+	insert into secao_encaminhamento_aee (questionario_id, nome, etapa, ordem, criado_em, criado_por, criado_rf)
+		values(questionarioId, 'Parecer Coordenação', 2, 1, NOW(), 'SISTEMA', '0');	
+
+	-- 1 - Quais mediações pedagógicas você realizou junto ao professor de classe comum antes de encaminhar o estudante ao AEE?
+	insert into questao(questionario_id, ordem, nome, observacao, obrigatorio, tipo, opcionais, criado_em, criado_por, criado_rf)
+		values(questionarioId, 1, 'Quais mediações pedagógicas você realizou junto ao professor de classe comum antes de encaminhar o estudante ao AEE?', '', true, 2, '', NOW(), 'SISTEMA', '0');
+
+	-- 2 - Além do professor de classe comum, o encaminhamento do estudante ao AEE foi discutido com outros profissionais da unidade educacional?
+	insert into questao(questionario_id, ordem, nome, observacao, obrigatorio, tipo, opcionais, criado_em, criado_por, criado_rf)
+		values(questionarioId, 2, 'Além do professor de classe comum, o encaminhamento do estudante ao AEE foi discutido com outros profissionais da unidade educacional?', '', true, 3, '', NOW(), 'SISTEMA', '0')
+		RETURNING id INTO questaoId;
+
+    -- 2.2.1 - Quais profissionais participaram desta discussão e quais as contribuições de cada um?
+	insert into questao(questionario_id, ordem, nome, observacao, obrigatorio, tipo, opcionais, criado_em, criado_por, criado_rf)
+		values(questionarioId, 0, 'Quais profissionais participaram desta discussão e quais as contribuições de cada um?', '', true, 2, '', NOW(), 'SISTEMA', '0')
```

Comment:
Esse item é de Ordem = 1 para gerar na descrição 2.1
```suggestion
		values(questionarioId, 1, 'Quais profissionais participaram desta discussão e quais as contribuições de cada um?', '', true, 2, '', NOW(), 'SISTEMA', '0')
```